arc-core/src/arc/struct/ObjectMap.java

```diff
@@ -183,6 +183,12 @@ public void putAll(ObjectMap<? extends K, ? extends V> map){
             put(entry.key, entry.value);
     }
 
+    public void putAll(Object... values) {
+        for(int i = 0; i < values.length / 2; i++){
+            put((K) values[i * 2], (V) values[i * 2 + 1]);
```

Comment:
redundant space after cast

Reply:
~~This was copy pasted from the code above~~

Reply:
well, then that's wrong too